<ui version="4.0" >
 <class>DigitalWindow</class>
 <widget class="QMainWindow" name="DigitalWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QLineEdit" name="noiseEdit" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>520</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="noiseLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>520</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Noise Amplitude</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>580</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="freqEdit" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>550</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeEdit" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>580</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="freqLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>550</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Frequency Offset</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>580</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Timing Offset</string>
    </property>
   </widget>
   <widget class="QFrame" name="sinkFrame" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1221</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1201</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="sinkLayout" />
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>closeButton</tabstop>
  <tabstop>noiseEdit</tabstop>
  <tabstop>freqEdit</tabstop>
  <tabstop>timeEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>DigitalWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>322</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel" >
     <x>66</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>DigitalWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>617</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
